### PR TITLE
[202012][show_bfd] add local discriminator in show bfd command

### PR DESCRIPTION
**Pull request by baorliu**
#### What I did
the pull request https://github.com/sonic-net/sonic-swss/pull/2587 added local_discriminator to state_db

In this pull request, local discriminator is added in the output of the following show commands :
```
show bfd summary
show bfd peer
```
the upcoming BFD debug CLI needs the local discriminator to specify BFD session.

#### How I did it
in show/main.py show bfd cli, read state db and show the value if there is local_discriminator field, and show "NA" if the field does not exist (backward compatible).

#### How to verify it
updated the test/show_bfd_test.py to test the local_discriminator field and show bfd cli output.
and also did manual test in a testbed (with modified bfdorch wich sonic-net/sonic-swss/pull/2587 diff ):
```
cisco@sonic:/var/tmp$ show bfd sum
Total number of BFD sessions: 1
Peer Addr    Interface    Vrf      State    Type          Local Addr      TX Interval    RX Interval    Multiplier  Multihop    Local Discriminator
-----------  -----------  -------  -------  ------------  ------------  -------------  -------------  ------------  ----------  ---------------------
192.85.2.24  default      default  Down     async_active  192.85.2.29            1000           1000             3  true        NA
cisco@sonic:/var/tmp$ show bfd peer 192.85.2.24
Total number of BFD sessions for peer IP 192.85.2.24: 1
Peer Addr    Interface    Vrf      State    Type          Local Addr      TX Interval    RX Interval    Multiplier  Multihop    Local Discriminator
-----------  -----------  -------  -------  ------------  ------------  -------------  -------------  ------------  ----------  ---------------------
192.85.2.24  default      default  Down     async_active  192.85.2.29            1000           1000             3  true        NA
cisco@sonic:/var/tmp$ docker exec -i swss swssconfig /dev/stdin < bfd.cfg
cisco@sonic:/var/tmp$ show bfd peer 192.85.2.24
Total number of BFD sessions for peer IP 192.85.2.24: 1
Peer Addr    Interface    Vrf      State    Type          Local Addr      TX Interval    RX Interval    Multiplier  Multihop      Local Discriminator
-----------  -----------  -------  -------  ------------  ------------  -------------  -------------  ------------  ----------  ---------------------
192.85.2.24  default      default  Down     async_active  192.85.2.29            1000           1000             3  true                            1
cisco@sonic:/var/tmp$ show bfd sum
Total number of BFD sessions: 1
Peer Addr    Interface    Vrf      State    Type          Local Addr      TX Interval    RX Interval    Multiplier  Multihop      Local Discriminator
-----------  -----------  -------  -------  ------------  ------------  -------------  -------------  ------------  ----------  ---------------------
192.85.2.24  default      default  Down     async_active  192.85.2.29            1000           1000             3  true                            1
```

#### Previous command output (if the output of a command-line utility has changed)
```
Peer Addr              Interface    Vrf      State    Type          Local Addr               TX Interval    RX Interval    Multiplier  Multihop
---------------------  -----------  -------  -------  ------------  ---------------------  -------------  -------------  ------------  ----------
100.251.7.1            default      default  Up       async_active  10.0.0.1                         300            500             3  true
fddd:a101:a251::a10:1  default      default  Down     async_active  fddd:c101:a251::a10:2            300            500             3  true
10.0.1.1               default      default  DOWN     async_active  10.0.0.1                         300            500             3  true
10.0.2.1               Ethernet12   default  UP       async_active  10.0.0.1                         200            600             3  false
2000::10:1             default      default  UP       async_active  2000::1                          100            700             3  false
10.0.1.1               default      VrfRed   UP       async_active  10.0.0.1                         400            500             5  false
```
#### New command output (if the output of a command-line utility has changed)
```
Peer Addr              Interface    Vrf      State    Type          Local Addr               TX Interval    RX Interval    Multiplier  Multihop    Local Discriminator
---------------------  -----------  -------  -------  ------------  ---------------------  -------------  -------------  ------------  ----------  ---------------------
100.251.7.1            default      default  Up       async_active  10.0.0.1                         300            500             3  true        NA
fddd:a101:a251::a10:1  default      default  Down     async_active  fddd:c101:a251::a10:2            300            500             3  true        NA
10.0.1.1               default      default  DOWN     async_active  10.0.0.1                         300            500             3  true        NA
10.0.2.1               Ethernet12   default  UP       async_active  10.0.0.1                         200            600             3  false       88
2000::10:1             default      default  UP       async_active  2000::1                          100            700             3  false       NA
10.0.1.1               default      VrfRed   UP       async_active  10.0.0.1                         400            500             5  false       NA
```

### CODE REVIEW

File: show/main.py [hunk 1]
```diff
@@ -1593,8 +1593,10 @@ def summary(db):
         for key in bfd_keys:
             key_values = key.split('|')
             values = db.db.get_all(db.db.STATE_DB, key)
+            if "local_discriminator" not in values.keys():
+                values["local_discriminator"]="NA"
```
Comment (siqbal1986) [inline show/main.py:1597]: spaces before and after =

File: show/main.py [hunk 2]
```diff
@@ -1622,8 +1624,10 @@ def peer(db, peer_ip):
         for key in bfd_keys:
             key_values = key.split(delimiter)
             values = db.db.get_all(db.db.STATE_DB, key)
+            if "local_discriminator" not in values.keys():
+                values["local_discriminator"]="NA"
```
Comment (siqbal1986) [inline show/main.py:1628]: spaces

**Review by siqbal1986 — COMMENTED** [review]
LGTM

File: show/main.py (hunk 1 above)
Comment (baorliu) [reply show/main.py]: updated. thx

File: show/main.py (hunk 2 above)
Comment (baorliu) [reply show/main.py]: updated. thx

**Review by siqbal1986 — APPROVED** [review]

**Pull request merged**